config/google_cloud_exporter/hostmetrics/README.md

```diff
@@ -0,0 +1,44 @@
+# Host Metrics with Google Cloud
+
+The Host Metrics Receiver can be used to send system metrics from an agent host to Google Cloud Monitoring. See the [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver) for more details.
+
+## Prerequisites
+
+See the [prerequisites](../prerequisites.md) doc for Google Cloud prerequisites.
+
+## Configuration
+
+An example configuration is located [here](./config.yaml).
+
+1. Copy [config.yaml](./config.yaml) to `/opt/observiq-otel-collector/config.yaml`
```

Comment:
These instructions are all linux specific I think we either need to be generic by saying `copy to your config location` or have a line for each OS's default location.

Reply:
My intention is to **not** ship these configs. Perhaps `docs/examples` is a more appropriate location. 

The current examples might work in any environment, but future examples may not. I do not want to give off the impression that they are "plug and play". Additionally, I can see the number of examples growing to 20-30, maybe 50.

Reply:
I think it's fine to keep these here for now. I think this is a cross that bridge when we come to it. If someone asks about windows stuff we address it then.